front/src/applications/operationalStudies/Home.tsx

```diff
@@ -31,12 +33,23 @@ export default function HomeOperationalStudies() {
   const { t } = useTranslation('operationalStudies/home');
   const safeWord = useSelector(getUserSafeWord);
   const [sortOption, setSortOption] = useState<SortOptions>('LastModifiedDesc');
-  const [projectsList, setProjectsList] = useState<
-    Array<ProjectWithStudies | SearchResultItemProject>
-  >([]);
-  const [selectedProjectIds, setSelectedProjectIds] = useState<number[]>([]);
   const [filter, setFilter] = useState('');
   const [filterChips, setFilterChips] = useState('');
+  const [deleteProject] = osrdEditoastApi.endpoints.deleteProjectsByProjectId.useMutation();
+
+  const {
+    selectedItemIds: selectedProjectIds,
+    setSelectedItemIds: setSelectedProjectIds,
+    items: projectsList,
+    setItems: setProjectsList,
+    toggleSelection: toggleProjectSelection,
+    deleteItems: deleteProjects,
+  } = useMultiSelection<ProjectWithStudies | SearchResultItemProject>((projectId) => {
+    if (selectedProjectIds.length > 0) {
+      deleteProject({ projectId });
+    }
+  });
```

Comment:
As you have done in the case of Scenario or Study. You can define a function which call the deleteProjects and check thid condition. And here, in `useMultiSelection` parameter, you pass only `deleteProject({ projectId })`